### ISSUE TITLE
HHVM support

### ISSUE THREAD

**Issue by nikolay** [awslabs/aws-elasticache-cluster-client-memcached-for-php#2]
Is HHVM supported?

**Comment by QuChen88** [comment]
I remember testing the library on HHVM a while back. You need to compile the libmemcached library (aws-elasticache-cluster-client-libmemcached) and compile HHVM on top of that.

**Comment by nikolay** [comment]
@QuChen88 Of course, there's a way to hack this (pun intended), but not one most PHP developers will take. So, developers will either drop HHVM (less probable due to the huge benefits) or drop ElastiCache. Given there's no PHP 7 support either, there's no alternative to HHVM at the moment. So, that's why Amazon should do their contribution to the project as Facebook has less incentive.

https://github.com/facebook/hhvm/issues/6702

**Comment by nikolay** [comment]
@QuChen88 Any update?

**Comment by tdlm** [comment]
:+1:

**Comment by vaibhu** [comment]
Do we have any update on the above request for HHVM support?

@nikolay do you mind sharing your solution - thx
